impeller/entity/contents/solid_fill_utils.cc

```diff
@@ -4,8 +4,35 @@
 
 #include "impeller/entity/contents/solid_fill_utils.h"
 
+#include "impeller/geometry/path.h"
+#include "impeller/renderer/device_buffer.h"
+#include "impeller/renderer/host_buffer.h"
+#include "impeller/tessellator/tessellator.h"
+
 namespace impeller {
 
-//
+VertexBuffer CreateSolidFillVertices(std::shared_ptr<Tessellator> tessellator,
+                                     const Path& path,
+                                     HostBuffer& buffer) {
+  VertexBuffer vertex_buffer;
+  VertexBuffer* vertex_buffer_out = &vertex_buffer;
```

Comment:
I'm sure there is a better way to write this buy my cpp-isms are weak

Reply:
I'm not clear on why `&vertex_buffer` followed by `vertex_buffer.vertex_buffer = ` int he closure wouldn't work...

Reply:
Some sort of stack allocation scoping issue. Not sure exactly, but if I do that then the buffer views don't exist anymore when I append to the command.

Reply:
Ahh, nah - fixed